pkg/csi/service/vanilla/controller_test.go

```diff
@@ -401,6 +418,76 @@ func TestCreateVolumeWithStoragePolicy(t *testing.T) {
 	}
 }
 
+//For this test, when the testbed has multiple shared datastores
+// but VC user which is usded to deploy CSI does not have Datastore.FileManagement privilege on
+// all shared datastores, the create volume should succeed.
+// This test is to simulate CSI on VMC.
+func TestCreateVolumeWithMultipleDatastores(t *testing.T) {
```

Comment:
If this test runs successfully, can you update the PR with the results?

Reply:
Yes. Update the test in the PR.